### PR TITLE
Assignment 4

### PR DESCRIPTION
**Pull request by jonwenz1**

### CODE REVIEW

File: Assignment4/jonwenz1-230786/task06.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+# Visualize the results
+g.add((ns.Researcher, RDFS.subClassOf, ns.Person))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.3: Create a new individual of Researcher named "Jane Smith"**"""
+
+# TO DO
+# Visualize the results
+g.add((ns.JaneSmith, ns.Researcher, Literal("Jane Smith")))
```
Comment (SerPablo) [inline Assignment4/jonwenz1-230786/task06.py:53]: Error, you are creating an individual,   Jane   ... type .... Researcher

File: Assignment4/jonwenz1-230786/task07.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+g.parse(github_storage+"/rdf/example6.rdf", format="xml")
+
+"""**TASK 7.1: List all subclasses of "LivingThing" with RDFLib and SPARQL**"""
+
+# TO DO
+from rdflib.plugins.sparql import prepareQuery
+q1 = prepareQuery('''
+    SELECT ?subClass
+    WHERE {
+        ?subClass rdfs:subClassOf* ns:LivingThing .
```
Comment (SerPablo) [inline Assignment4/jonwenz1-230786/task07.py:31]: With + to not show LivingThing

File: Assignment4/jonwenz1-230786/task07.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+"""**TASK 7.4:  List the name of the persons who know Rocky**"""
+
+# TO DO
+from rdflib import FOAF
+from rdflib import XSD
+VCARD = Namespace("http://www.w3.org/2001/vcard-rdf/3.0/")
+
+q4 = prepareQuery('''
+    SELECT ?person
+    WHERE {
```
Comment (SerPablo) [inline Assignment4/jonwenz1-230786/task07.py:87]: Only those that are Persons

File: Assignment4/jonwenz1-230786/task07.py [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+"""First let's read the RDF file"""
+
+from rdflib import Graph, Namespace, Literal
+from rdflib.namespace import RDF, RDFS
+g = Graph()
+g.namespace_manager.bind('ns', Namespace("http://somewhere#"), override=False)
+g.namespace_manager.bind('vcard', Namespace("http://www.w3.org/2001/vcard-rdf/3.0#"), override=False)
+g.parse(github_storage+"/rdf/example6.rdf", format="xml")
+
```
Comment (SerPablo) [inline Assignment4/jonwenz1-230786/task07.py:23]: All exercises have to be in SPARQL and RDFlib

**Review by SerPablo — COMMENTED** [review]
Review comments for task 6,7, Do the pull justo for Assignment 4

**Comment by anaigmo** [comment]
The PR is conflictive and contains more files than the specified for the assignment, closing without merge

**Pull request closed without merging**